packages/js/src/Modules/Verto/BrowserSession.ts

```diff
@@ -201,6 +201,28 @@ export default abstract class BrowserSession extends BaseSession {
 
   /**
    * Return the device list supported by the browser
```

Comment:
I think this reads slightly better 
```suggestion
   * Returns a list of devices supported by the browser
```